cli/popper/runner_host.py

```diff
@@ -181,13 +186,13 @@ def _get_build_info(self, step):
         return (build, img, tag, build_source)
 
     def _create_container(self, cid, step):
-        build, img, tag, dockerfile = self._get_build_info(step)
+        build, img, tag, build_source = self._get_build_info(step)
```

Comment:
@ivotron Actually, it is the path to the parent dir that contains the `Dockerfile` and potentially other script files like `entrypoint.sh`, `README.md`, etc i-e the context.  That's why I thought to call it only `dockerfile` might look like `/path/to/context/Dockerfile` instead of `/path/to/context` when the latter is used by `docker build`. What do you think ?

Reply:
yeah, I thought of `dockerfile_path`, but maybe we use `build_context` instead, as this would be in line with Docker's terminology (see [`build` help](https://docs.docker.com/engine/reference/commandline/build/)).

Reply:
Yeah, sounds good.